packages/edit-post/src/editor.js

```diff
@@ -65,6 +65,11 @@ function Editor( {
 						? getEditedPostTemplate()
 						: null,
 				post: postObject,
+				defaultRenderingMode:
+					currentPost.postType === 'wp_template' ||
+					currentPost.postType === 'wp_template_part'
+						? 'all'
```

Comment:
This mode has been removed in https://github.com/WordPress/gutenberg/pull/58935 are we restoring it back? Why?

Reply:
Fixed in 120c78555b8560bba67d3db5349e10a39916233f.

Also updated some docs missed in https://github.com/WordPress/gutenberg/pull/58935: c54239624a160cb4881012881775cacef095bbb3.